src/vmm/src/devices/virtio/block/device.rs

```diff
@@ -15,25 +15,28 @@ use crate::devices::virtio::queue::Queue;
 use crate::devices::virtio::{ActivateError, TYPE_BLOCK};
 use crate::rate_limiter::BucketUpdate;
 use crate::snapshot::Persist;
+use crate::vmm_config::drive::BlockDeviceConfig;
```

Comment:
commit `refactor(block): construct Block from BlockDeviceConfig`: can add more info on why this is simpler.

Reply:
updated commit message